### PR TITLE
[cRO] fix freeze char

### PR DESCRIPTION
**Pull request by alisonrag**
fix freeze char when change map.

merge map_change and map_changed

### CODE REVIEW

File: src/Network/Receive/ServerType0.pm [hunk 1]
```diff
@@ -623,6 +623,7 @@ sub new {
 		'0AA2' => ['refineui_info', 'v v C a*' ,[qw(len index bless materials)]],
 		'0AC4' => ['account_server_info', 'v a4 a4 a4 a4 a26 C x17 a*', [qw(len sessionID accountID sessionID2 lastLoginIP lastLoginTime accountSex serverInfo)]],
 		'0AC5' => ['received_character_ID_and_Map', 'a4 Z16 a4 v a128', [qw(charID mapName mapIP mapPort mapUrl)]],
+
```
Comment (sctnightcore) [inline src/Network/Receive/ServerType0.pm]: well :thinking:

Comment (alisonrag) [reply src/Network/Receive/ServerType0.pm]: 0ac5 and 0ac7 is already implemented in st0/sakexe_0

Comment (sctnightcore) [reply src/Network/Receive/ServerType0.pm]: It has a gap

Comment (sctnightcore) [reply src/Network/Receive/ServerType0.pm]: It has a gap

**Pull request merged**